src/Multiplicator.java

```diff
@@ -0,0 +1,25 @@
+import java.util.Scanner;
+
+// User should input a positive integer and receive multiplications up to x9
+public class Multiplicator {
+    public static void main(String[] args) {
+        Scanner input = new Scanner(System.in);
+        int userInput = input.nextInt();
+        int multiplicative;
+
+        /* if (!input.hasNextInt()) {
+            System.out.println("Please insert positive integer! "); //Check if entered value is int

```

Comment:
qo mot chi ashxatel, vorovhetev 'int userInput = input.nextInt();' es toxov du arden isk Int es uzel, el heto el inch es check anum :)

es kargi ban es uzel erevi :)  

Scanner input = new Scanner(System.in);
        int userInput = 0;

        if (!input.hasNextInt()) {
            System.out.println("Please insert positive integer! "); //Check if entered value is int
        } else {
            userInput = input.nextInt();
        }